hugo/content/010-blog/2018_week_10/_index.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Shared storage with S3 backend
+type: Blog
+---
+
+The storage is definitely the most complex and important part of an application setup, once this part is completed, 
+one of the most problematic parts could be solved.
+
+Mounting a S3 bucket into a pod using [FUSE](https://github.com/libfuse/libfuse) allows to access data stored in S3 via 
+the filesystem. The mount is a pointer to an S3 location, so the data is never synced locally. Once mounted, any pod 
+can read or even write from that directory without the need for explicit keys.
+
+
+{{% blog_img "logo" "blog-s3-shared-storage.png" %}}
+
+
+However, it can be used to import and parse large amounts of data into a database.
+
+..read on [Shared S3 Storage]({{ site.baseurl }}/doc/2017/01/16/app-s3.html) how to configure it.
```

Comment:
this expression creates a broken link. If locally served the link looks like this
http://localhost:1313/blog/%7B%7B%20site.baseurl%20%7D%7D/doc/2017/01/16/app-s3.html

Reply:
0f51bff7c3940280e56549044adcefcb01dc3ff8